pattr.py

```diff
@@ -216,6 +216,18 @@ def send_room_message(message):
                  {'data': message, 'sender': session['nick']},
                  room=session['room'])
 
+        elif ('www.' in message['data'] and ('.com' in message['data'] or '.net' in message['data'] or '.edu' in message['data'] or '.gov' in message['data']
+        or '.io' in message['data'] or '.uk' in message['data'] or '.ca' in message['data'] or '.de' in message['data'] or '.fr' in message['data'] or '.us' in message['data']
+        or '.it' in message['data'] or '.biz' in message['data'] or '.xyz' in message['data'] or '.co' in message['data'] or '.me' in message['data'])):
```

Comment:
In order to make this a little bit more readable/maintainable, let's add each domain extension into a list, so then instead of a bunch of statements to check if the extension is in the domain. I'm thinking it would look something like this:
```
domain_extensions = ['.com', '.net', '.edu', '.gov', '.io', '.uk', '.ca', '.de', '.fr', '.us', '.it', '.biz', '.xyz', '.co', '.me']

elif 'www.' in message['data'] and any(ext in message['data'] for ext in domain_extensions):
    # do something
```